src/pages/index.js

```diff
@@ -8,62 +8,204 @@ import {
   selectors,
   profileTitleInput,
   profileDescriptionInput,
+  avatarModalFormSelector,
+  editButtonAvatar,
+  avatarSelector,
+  cardDeleteModal,
 } from "../utils/constants.js";
 import UserInfo from "../components/UserInfo.js";
 import FormValidator from "../components/FormValidator.js";
+import Api from "../components/Api.js";
+
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/group-12",
+  authToken: "cb447498-fb2c-4c99-9fc5-8ee58bc7fe4c",
+});
 
 // Popup with Image
 const cardPreview = new PopupWithImage({
   popupSelector: "#preview-image-modal",
 });
 cardPreview.setEventListeners();
 
-// Section / Card
-
+// Section / Card Create Card
 const renderCard = (data) => {
+  const likes = data.likes || [];
   const cardElement = new Card(
     {
       data,
       handleImageClick: (imageData) => {
         cardPreview.open(imageData);
       },
+      handleDeleteClick: () => {
+        deleteModal.open();
+        deleteModal.setSubmitAction(() => {
+        deleteModal.renderLoading(true);
```

Comment:
It's important to format code correctly, use the indentation hierarchy, and split your code. Please check these sources to study the standard: [https://code.s3.yandex.net/web-developer/landings/design-rules-en/index.html](https://code.s3.yandex.net/web-developer/landings/design-rules-en/index.html), [https://javascript.info/coding-style](https://javascript.info/coding-style).

To make sure your code is formatted correctly, we suggest using Prettier (https://prettier.io/). It can be integrated with any code editor or IDE.

You should fix the formatting in all the files

Reply:
Done